lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
+ * @class
+ * @returns {Menu}
+ */
+var Menu = (function () {
+
+    Menu.prototype = new Control();
```

Comment:
The JSDoc doclet will need to say `@alias Menu` so that JSDoc knows that it's members are a part of the final Menu class.